doc/rados/operations/monitoring-osd-pg.rst

```diff
@@ -50,129 +50,128 @@ not assign placement groups to the OSD. If an OSD is ``down``, it should also be
 - An OSD was ``down``, was restarted, and is now ``recovering``.
-- An OSD in the Acting Set is ``down`` or unable to service requests, 
+- An OSD in the Acting Set is ``down`` or unable to service requests,
   and another OSD has temporarily assumed its duties.
 
-In most cases, the Up Set and the Acting Set are identical. When they are not,
-it may indicate that Ceph is migrating the PG (it's remapped), an OSD is
-recovering, or that there is a problem (i.e., Ceph usually echoes a "HEALTH
-WARN" state with a "stuck stale" message in such scenarios).
+Typically, the Up Set and the Acting Set are identical to each other. When they are not,
+it might indicate that Ceph is migrating the PG (in other words, that the PG has been remapped), that an OSD is
+recovering, or that there is a problem with the cluster (in such scenarios, Ceph usually echoes a "HEALTH
+WARN" state with a "stuck stale" message).
 
-To retrieve a list of placement groups, execute:
+To retrieve a list of PGs, run the following command:
 
 .. prompt:: bash $
 
-	ceph pg dump
-	
-To view which OSDs are within the Acting Set or the Up Set for a given placement
-group, execute:
+    ceph pg dump
+
+To see which OSDs are within the Acting Set and the Up Set for a specific PG, run the following command:
 
 .. prompt:: bash $
 
-	ceph pg map {pg-num}
+    ceph pg map {pg-num}
 
-The result should tell you the osdmap epoch (eNNN), the placement group number
-({pg-num}), the OSDs in the Up Set (up[]), and the OSDs in the acting set
+The output provides the following information: the osdmap epoch (eNNN), the PG number
+({pg-num}), the OSDs in the Up Set (up[]), and the OSDs in the Acting Set
 (acting[])::
 
-	osdmap eNNN pg {raw-pg-num} ({pg-num}) -> up [0,1,2] acting [0,1,2]
+    osdmap eNNN pg {raw-pg-num} ({pg-num}) -> up [0,1,2] acting [0,1,2]
 
-.. note:: If the Up Set and Acting Set do not match, this may be an indicator
-   that the cluster rebalancing itself or of a potential problem with 
+.. note:: If the Up Set and the Acting Set do not match, this might indicate
+   that the cluster is rebalancing itself or that there is a problem with 
    the cluster.
- 
+
 
 Peering
 =======
 
-Before you can write data to a placement group, it must be in an ``active``
-state, and it **should** be in a ``clean`` state. For Ceph to determine the
-current state of a placement group, the primary OSD of the placement group
-(i.e., the first OSD in the acting set), peers with the secondary and tertiary
-OSDs to establish agreement on the current state of the placement group
-(assuming a pool with 3 replicas of the PG).
-
+Before you can write data to a PG, it must be in an ``active``
+state and it will preferably be in a ``clean`` state. For Ceph
+to determine the current state of a PG, peering must take place.
+That is, the primary OSD of the PG (that is, the first OSD in
+the Acting Set) must peer with the secondary and tertiary OSDs
+so that agreement on the current state of the PG can be
```

Comment:
s/agreement/consensus/

because it sounds cooler

Reply:
It does have a parliamentary connotation that "agreement" lacks.